pages/apps/details/[appDetails].tsx

```diff
@@ -0,0 +1,34 @@
+import { GetStaticPaths, GetStaticProps } from 'next'
+import ApplicationDetails from '../../../src/components/application/Details'
+import Main from '../../../src/components/layout/Main'
+
+export default function Details({ data }) {
+  return (
+    <Main>
+      <ApplicationDetails data={data} />
+    </Main>
+  )
+}
+
+export const getStaticProps: GetStaticProps = async ({params: {appDetails}}) => {
+  console.log("Fetching data for app details: ", appDetails)
+  const appData = await fetch(`https://flathub-backend.openshift.gnome.org/master/appstream/${appDetails}`)
```

Comment:
I would avoid hardcoding the server URL here

Reply:
Sure, nice catch, I must have missed it :)